src/tools/compiletest/src/runtest.rs

```diff
@@ -3124,15 +3124,17 @@ impl<'test> TestCx<'test> {
     fn check_mir_dump(&self) {
         let test_file_contents = fs::read_to_string(&self.testpaths.file).unwrap();
 
-        let mut test_dir = self.testpaths.file.with_extension("");
+        let test_dir = self.testpaths.file.parent().unwrap();
 
+        let mut bit_width = "all";
```

Comment:
keep this string empty and prepend a `.` to the string if there's an actual bit width. This way we have shorter names in the regular case

Reply:
👍